app/controllers/react/settings_controller.rb

```diff
@@ -0,0 +1,7 @@
+module React
```

Comment:
Also here I will name this stuff something different than react maybe Client API or jsAPI ....